cypress/features/regression/showEditPodClassic.feature

```diff
@@ -134,7 +134,7 @@ Feature: Show Edit Pod component
   Scenario Outline: Set Show Edit Pod buttonAlign to <buttonAlign>
     When I select buttonAlign to "<buttonAlign>"
       And I edit Show Edit Pod component
-    Then Show Edit Pod buttonAlign value is set to "<buttonAlign>"
+    Then Show Edit Pod buttons are aligned <buttonAlign>
```

Comment:
Grammatically correct should be "Show Edit Pod buttons are aligned to <buttonAlign>"

Reply:
fixed